source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -1136,7 +1137,13 @@ TEST_F(NiDAQmxDriverApiTests, GetErrorString_ReturnsErrorMessage)
   auto status = get_error_string(DAQmxErrorInvalidAttributeValue, response);
 
   EXPECT_SUCCESS(status, response);
-  EXPECT_THAT(response.error_string(), HasSubstr("Requested value is not a supported value for this property."));
+  const auto& error_string = response.error_string();
```

Comment:
TODO: not sure why this is in the diff b/c this is already in. Make sure it's fixed when remerging.